social-wall/js/social_wall.js

```diff
@@ -1,7 +1,7 @@
-$(document).keypress(function(event){
+$(document).keypress(function (event) {
 
     var keycode = (event.keyCode ? event.keyCode : event.which);
-    if(keycode == '13'){
+    if(keycode == '13') {
         $('#search-but').click();   
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/social-wall/js/social_wall.js
+++ b/social-wall/js/social_wall.js
@@ -2,7 +2,7 @@
 
     var keycode = (event.keyCode ? event.keyCode : event.which);
     if(keycode == '13') {
-        $('#search-but').click();   
+        $('#search-but').click();
     }
 
 });
```